### PR TITLE
fix: Storybook setup

### PR DESCRIPTION
**Pull request by ff6347**

### CODE REVIEW

File: src/components/Sidebar/Sidebar.stories.tsx [hunk 1]
```diff
@@ -149,14 +148,14 @@ AuthRecovery.args = {
...
+// AuthReset.args = {
+//   match: '/auth',
+//   children: (
+//     <PasswordResetForm
+//       returnClickHandler={() => undefined}
+//       additionalSubmitHandler={() => undefined}
+//       setNotification={() => undefined}
+//     />
+//   ),
+// };
```
Comment (raphael-arce) [inline src/components/Sidebar/Sidebar.stories.tsx:161]: can this be removed?

File: .storybook/preview.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+// const OriginalNextImage = NextImage.default;
 
-Object.defineProperty(NextImage, 'default', {
-  configurable: true,
-  value: props => <OriginalNextImage {...props} unoptimized />,
-});
+// Object.defineProperty(NextImage, 'default', {
+//   configurable: true,
+//   value: props => <OriginalNextImage {...props} unoptimized />,
+// });
```
Comment (raphael-arce) [inline .storybook/preview.js:9]: can this be removed?

**Review by raphael-arce — APPROVED** [review]
Couple of code comments that could maybe be removed, but other lgtm 👍  thx

**Comment by Jaszkowic** [comment]
I close this because it is outdated and we don't use Storybook anymore.

**Pull request closed without merging**